distros/raptor/remote.list

```diff
@@ -0,0 +1,2 @@
+shared-worker-memory,https://raptor-shared-worker-memory.github.io/package.webapp
```

Comment:
These are examples, still I didn't find any good and non-hacky way to host several apps on the same domain with fixed ```origin``` that Raptor can successfully run - so for now separate subdomains.